src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -51,6 +51,15 @@ Behavior:
   ValuedEvents:
     - name: tuningTones
       description: Times at which tuning tones are played to an animal after a behavioral experiment during ephys recording sessions.
+  VideoCamera1:
```

Comment:
I am confused here, there are three cameras and two elements on the list here, does it mean six in total?

Are you missing one?

And another question: what are the units by default in the VideoInterface? I am asking because you are adding Frames here and I don't remember ever seeing this.

Reply:
> I am confused here, there are three cameras and two elements on the list here, does it mean six in total?
> Are you missing one?

I believe I am missing 1 and/or the description is incorrect.  I think there are _up to three_ cameras recording each session, but since I only have 1 example session I'm not totally sure.  I'll add to list of questions tho.

Reply:
> And another question: what are the units by default in the VideoInterface? I am asking because you are adding Frames here and I don't remember ever seeing this.

The default units is Frames in the `get_metadata` method of `VideoInterface`, so I'm just replicating that here.